struct-patch-derive/src/lib.rs

```diff
@@ -100,6 +100,27 @@ impl Patch {
         #[cfg(not(feature = "status"))]
         let patch_status_impl = quote!();
 
+        let add_impl = quote! {
+            impl core::ops::Add<#name #generics> for #struct_name #generics #where_clause {
+                type Output = #struct_name;
```

Comment:
You'd better use `Self`:

```suggestion
                type Output = Self;
```